kevm-pyk/src/kevm_pyk/kproj/evm-semantics/driver.md

```diff
@@ -382,14 +382,14 @@ Note that `TEST` is sorted here so that key `"network"` comes before key `"pre"`
  // ---------------------------------------
     rule <k> #halt ~> check J:JSON => check J ~> #halt ... </k>
 
-    rule <k> check DATA : { .JSONs } => .K ... </k> requires DATA =/=String "transactions"
-    rule <k> check DATA : [ .JSONs ] => .K ... </k> requires DATA =/=String "ommerHeaders"
+    rule <k> check DATA : { .JSONs } => .K ... </k> requires notBool DATA in (SetItem("transactions") SetItem("withdrawals"))
```

Comment:
Given the size of the set, probably the impact is null, but wouldn't it be more efficient to evaluate a side condition like `DATA =/=String "transactions orBool DATA =/=String "withdrawals"` instead of a set-based one?

Reply:
Actualy, I'm not sure how big is the overhead caused by using `Set` here, but given that `driver.md` is only used for the conformance test suite using the llvm backend, I'd say it wouldn't make a difference. I used it here to be consistent with already existing code ([sample](https://github.com/runtimeverification/evm-semantics/blob/master/kevm-pyk/src/kevm_pyk/kproj/evm-semantics/driver.md?plain=1#L389)).